File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_AutoTextGroupsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_AutoTextGroupsService : BaseService<Sys_AutoTextGroups, ISys_AutoTextGroupsRepository>
    , ISys_AutoTextGroupsService
    {
    private readonly ISys_AutoTextGroupsRepository _Sys_AutoTextGroupsRepository;
    public Sys_AutoTextGroupsService(ISys_AutoTextGroupsRepository Sys_AutoTextGroupsRepository) : base(Sys_AutoTextGroupsRepository)
    {
    _Sys_AutoTextGroupsRepository = Sys_AutoTextGroupsRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_AutoTextGroups.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_AutoTextGroups } from './Sys_AutoTextGroups.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_AutoTextGroupsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'AutoText_Id', 'Group_Id'];

    List: Sys_AutoTextGroups[] | undefined;
    ListFilter: Sys_AutoTextGroups[] | undefined;
    FormData!: Sys_AutoTextGroups;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_AutoTextGroups";
    }
}

